package.json

```diff
@@ -14,8 +15,9 @@
     "inter": "^2.1.0",
     "markdown-it-mathjax3": "^4.3.2",
     "markdown-it-vuepress-code-snippet-enhanced": "github:IMB11/md-it-enhanced-snippets#dfb9fa2",
-    "vitepress": "^1.0.0-rc.33",
-    "vitepress-versioning-plugin": "^0.10.0",
-    "vue": "^3.4.0"
+    "patch-package": "^8.0.0",
```

Comment:
is this not a devDependencies?

Reply:
Potentially? I didn't test if it worked as a devDependency